src/static/js/modules/util/assign.js

```diff
@@ -1,5 +1,5 @@
 /* ==========================================================================
-   Shallow Extends
+   Assign
```

Comment:
Should we include the original name and link to the repo so folks can find the original source code?